static/skin/taskbar.js

```diff
@@ -3,46 +3,51 @@ function htmlDecode(input) {
     return doc.documentElement.textContent;
 }
 
-const jq = jQuery.noConflict(true);
-jq(document).ready(() => {
-    (function ($) {
-        const root = $( `link[type='root']` ).attr("href");
+function on_ready() {
```

Comment:
This named functions asks for a conflict with javascript code coming from the ZIM file. Please use an unnamed function.